app/views/shops/orders.html.erb

```diff
@@ -14,8 +14,8 @@
   <div class="col s5 pagination-holder">
     <ul class="tabs no-border-bottom">
       <li class="tab col s3"><a href="" >All (<%= all_orders_count %>)</a></li>
-      <li class="tab col s3"><a  href="?status=completed">Completed (<%= Order.pending(@shop).count %>)</a></li>
-      <li class="tab col s3"><a href="?status=pending" >Pending (<%= Order.completed(@shop).count %>)</a></li>
+      <li class="tab col s3"><a  href="?status=completed">Completed (<%= Order.completed(@shop).count %>)</a></li>
```

Comment:
@andela-anwocha You should move the logic outside the view, you could make this a variable in the controller method that you can then access in the view